common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -73,7 +68,7 @@ define('video/04_video_full_screen.js', [], function () {
         state.videoFullScreen.fullScreenEl = $(template);
         state.videoFullScreen.sliderEl = state.el.find('.slider');
         state.videoFullScreen.fullScreenState = false;
-        state.el.find('.secondary-controls').append(state.videoFullScreen.fullScreenEl);
+        HtmlUtils.append(state.el.find('.secondary-controls'), HtmlUtils.HTML(state.videoFullScreen.fullScreenEl));
         state.videoFullScreen.updateControlsHeight();
     }
```

Comment:
Make sure this JS is referencing a control-text element that still exists (it might... I just can't tell by looking at the code).

````
function enter() {
        var fullScreenClassNameEl = this.el.add(document.documentElement),
            closedCaptionsEl = this.el.find('.closed-captions');

        this.scrollPos = $(window).scrollTop();
        $(window).scrollTop(0);
        this.videoFullScreen.fullScreenState = this.isFullScreen = true;
        fullScreenClassNameEl.addClass('video-fullscreen');
        this.videoFullScreen.fullScreenEl
            .find('.icon')
                .removeClass('fa-arrows-alt')
                .addClass('fa-compress')
                .find('.control-text')
                    .text(gettext('Exit full browser'));

        this.el.trigger('fullscreen', [this.isFullScreen]);

        $(closedCaptionsEl).css({
            'top': '70%',
            'left': '5%'
        });
    }